scripts/prepare_sector_network.py

```diff
@@ -2347,17 +2348,394 @@ def add_biomass(n, costs):
         )
 
 
-def add_industry(n, costs):
-    logger.info("Add industrial demand")
+def add_low_t_industry(n, nodes, industrial_demand, costs, must_run):
+    """
+    Add low temperature heat for industry.
+    """
 
-    nodes = pop_layout.index
-    nhours = n.snapshot_weightings.generators.sum()
-    nyears = nhours / 8760
+    logger.info("Add low temperature industry.")
+
+    n.madd(
+        "Bus",
+        nodes + " lowT industry",
+        location=nodes,
+        carrier="lowT industry",
+        unit="MWh_LHV",
+    )
+
+    n.madd(
+        "Load",
+        nodes,
+        suffix=" lowT industry",
+        bus=nodes + " lowT industry",
+        carrier="lowT industry",
+        p_set=industrial_demand.loc[nodes, "solid biomass"] / 8760.0,
+    )
+
+    if (
+        options["industry_t"]["low_T"]["biomass"]
+        or not options["industry_t"]["endogen"]
+    ):
+        n.madd(
+            "Link",
+            nodes,
+            suffix=" solid biomass for lowT industry",
+            bus0=spatial.biomass.nodes,
+            bus1=nodes + " lowT industry",
+            bus2="co2 atmosphere",
+            carrier="lowT industry solid biomass",
+            p_nom_extendable=True,
+            p_min_pu=must_run,
+            efficiency=costs.at["solid biomass boiler steam", "efficiency"],
+            efficiency2=costs.at["solid biomass", "CO2 intensity"]
+            - costs.at["solid biomass", "CO2 intensity"],
+            capital_cost=costs.at["solid biomass boiler steam", "fixed"]
+            * costs.at["solid biomass boiler steam", "efficiency"],
+            marginal_cost=costs.at["solid biomass boiler steam", "VOM"],
+        )
+
+        n.madd(
+            "Link",
+            nodes,
+            suffix=" solid biomass for lowT industry CC",
+            bus0=spatial.biomass.nodes,
+            bus1=nodes + " lowT industry",
+            bus2=spatial.co2.nodes,
+            bus3="co2 atmosphere",
+            carrier="lowT industry solid biomass CC",
+            p_nom_extendable=True,
+            p_min_pu=must_run,
+            efficiency=costs.at["solid biomass boiler steam CC", "efficiency"],
+            capital_cost=costs.at["solid biomass boiler steam CC", "fixed"]
+            * costs.at["solid biomass boiler steam CC", "efficiency"]
+            + costs.at["biomass CHP capture", "fixed"]
+            * costs.at["solid biomass", "CO2 intensity"],
+            marginal_cost=costs.at["solid biomass boiler steam CC", "VOM"],
+            efficiency3=costs.at["solid biomass", "CO2 intensity"]
+            * (1 - costs.at["biomass CHP capture", "capture_rate"])
+            - costs.at["solid biomass", "CO2 intensity"],
```

Comment:
same here: should be without the `- costs.at["solid biomass", "CO2 intensity"]`, shouldn't it?